docs/concepts/workloads/pods/init-containers.md

```diff
@@ -17,7 +17,7 @@ scripts not present in an app image.
 
 {:toc}
 
-{% include 1-5-beta.md %}
+{% assign for_k8s_version="v1.5" %}{% include feature-state-beta.md %}
 
 **Once the feature exits beta, Init Containers will be specified in the PodSpec
```

Comment:
Are you referring to the **FEATURE STATE** tag from the include or the additional text, "**Once the feature exits beta**..."? The latter was there from before, but I can change that too while I'm at it.